user/user_validator_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package user
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/clearlinux/clr-installer/utils"
+)
+
+func init() {
+	utils.SetLocale("en_US.UTF-8")
+}
+
+const (
+	cracklibpath = "/usr/bin/cracklib-check"
```

Comment:
Please move this const into user_validator and reference the const in place of the same string in that code.

Reply:
Done